common/include/Renderer.h

```diff
@@ -32,7 +32,18 @@ namespace Interfaces
     {
         /// General initialization for all the scenes implemented by the component
         SCENE_TYPE_TEST0,
-        SCENE_TYPE_TEST1,
+        //SCENE_TYPE_TEST1,
+		SCENE_SOLAR_SYSTEM1,  // This will be the main solar system scene
```

Comment:
Can you correct formatting? I can see new enums added shifted by tab. 

Also we can remove the numbers appended after the scene type? 

I know that the existing enums can create such confusions. 

I can see that you've commented test1 scene.  Was there a reason? 

The placeholder commit seems to be removed. 

Rest looks good to me.